install/helm/supergloo/templates/3-deployment.yaml

```
@@ -29,6 +29,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: SIDECAR_INJECTOR_IMAGE
+              value: "{{ .Values.supergloo.sidecarInjectorImage.repository }}:{{ .Values.supergloo.sidecarInjectorImage.tag }}"
```

what about this format instead?
```suggestion
              value: "{{ .Values.supergloo.sidecarInjectorImage.imageRpository }}/sidecar-injector:{{ .Values.supergloo.sidecarInjectorImage.tag }}"
```

---

The underlying type has the field `repository` not `imageRepository`
It's a common field among all of our generate scripts

---

Yes, Marco and I discussed changing the underlying spec. Since `sidecar-injector` is a "constant" it should not be changeable through the helm values.
I think Marco is planning on defining a new struct to use for this

---

But then that would be true for every single one of our containers. Currently we use this pattern across all of our projects

---

I'd rather have the image name in a  go constant and use that to build the image name in the chart generation script.

---

Oops, did not refresh the page and did not see the new comments. I thought more about it and I'd rather keep the whole image name (e.g.`quay.io/solo-io/supergloo`) as a helm value and concatenate the prefix and the image name in our chart generation script. I don't see the value of splitting the name in 2 in the values file (pun not intended).